images/template-service-broker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# This image will run the template service broker.
+# The entry point name is not yet final.
+
+FROM openshift/origin
+
+ENTRYPOINT ["/usr/bin/openshift start template-service-broker"]
```

Comment:
> do we really need a new image? why can't we use the origin image w/ an appropriate command defined in the pod?

Yeah, you should really have one because there's no reason for the the TSB to be built on openshift.